contrib/inventory/foreman.ini

```diff
@@ -123,13 +123,45 @@ user = foreman
 password = secret
 ssl_verify = True
 
+#Foreman 1.24 introduces a new reports API to improve performance of the inventory script. Set to False if you want to use the old API. Defaults to True.
```

Comment:
Nit: this relies on the `foreman_ansible` plugin. Users who have 1.24 but not the plugin will see an error and be confused. Plesae add it as a note. Also add a space after `#` and wrap the line for readability.

Reply:
Updated.